src/bluecadet/touch/drivers/MultiNativeTouchDriver.h

```diff
@@ -0,0 +1,184 @@
+// Thanks to code from TouchScript by Valentin Simonov / http://va.lent.in/
+// https://github.com/TouchScript/TouchScript/tree/master/External/WindowsTouch
+
+#pragma once
+
+#include <Windows.h>
+
+#include "cinder/app/App.h"
+#include "cinder/gl/gl.h"
+#include "cinder/System.h"
+#include "cinder/Signals.h"
+#include "cinder/Log.h"
+
+#include "../TouchManager.h"
+
+namespace bluecadet {
+namespace touch {
+namespace drivers {
+
+ci::signals::Signal< void(ci::vec2, int) > & getSignalOnTouchAdded();
```

Comment:
See notes above about folding this into the class itself; If that's the root of the DLL export error, let's review together.